<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updatestudent</class>
 <widget class="QDialog" name="updatestudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1262</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="logo.qrc">
    <normaloff>:/new/prefix1/images/Tools.png</normaloff>:/new/prefix1/images/Tools.png</iconset>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>1271</width>
     <height>731</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget{
border-image:url(:/image/main.png)
}</string>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>691</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#groupBox{
font-size : 25px;
background-color:rgba(0, 152, 0, 0.496);
border: 1px solid black;
border-radius: 10%;
}</string>
    </property>
    <property name="title">
     <string>UPDATES STUDENT'S INFO:</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Old English Text MT</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="text">
      <string>code :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Old English Text MT</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="text">
      <string>first name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Enter The Code of the student u wanna Update his infos</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Enter The new First Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>180</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Enter The new Last Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Old English Text MT</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="text">
      <string>last name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>240</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Enter the new class info</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Old English Text MT</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="text">
      <string>class:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>300</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Enter the new adress </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Old English Text MT</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="text">
      <string>adresse:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>360</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Enter the new password </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Old English Text MT</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="text">
      <string>password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>420</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Reenter the new password </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>410</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Old English Text MT</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>confirm password:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>480</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton{
background-color:#0d6efd;
border-radius: 10%;
}
#pushButton:hover{
background-color:#6610f2;
cursor: pointer;

}</string>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
     <property name="icon">
      <iconset resource="logo.qrc">
       <normaloff>:/new/prefix1/images/edit.png</normaloff>:/new/prefix1/images/edit.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>480</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_2{
background-color:#0d6efd;
border-radius: 10%;
}
#pushButton_2:hover{
background-color:#6610f2;
cursor: pointer;

}</string>
     </property>
     <property name="text">
      <string>close</string>
     </property>
     <property name="icon">
      <iconset resource="logo.qrc">
       <normaloff>:/new/prefix1/images/31631915367osnbo8bns2l5yxoye3vm0yqten7dpvr2dlsp7wwxbxtlnnyzkcqqsdi5tzivxuuiun7ybau4atrz7amnqozue37efb6vzeb5dzgt.png</normaloff>:/new/prefix1/images/31631915367osnbo8bns2l5yxoye3vm0yqten7dpvr2dlsp7wwxbxtlnnyzkcqqsdi5tzivxuuiun7ybau4atrz7amnqozue37efb6vzeb5dzgt.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>30</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arty Signature</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_3{
background-color:#0d6efd;
border-radius: 10%;
}
#pushButton_3:hover{
background-color:#6610f2;
cursor: pointer;

}</string>
    </property>
    <property name="text">
     <string>HOME</string>
    </property>
    <property name="icon">
     <iconset resource="logo.qrc">
      <normaloff>:/new/prefix1/images/375-3752606_homepage-icon-house-logo-png-white.png</normaloff>:/new/prefix1/images/375-3752606_homepage-icon-house-logo-png-white.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="logo.qrc"/>
  <include location="logo.qrc"/>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
